spring-boot/admin/src/main/resources/templates/admin/fragments/gateway_health_monitor.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:th="http://www.thymeleaf.org">
+
+
+<body>
+<div class="col-md-4 col-sm-4 col-xs-12">
+    <div class="x_panel">
```

Comment:
is this maybe un-used code? please check and clean up, if required.

Reply:
Done